src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingPassBase.cs

```diff
@@ -621,6 +638,18 @@ bool IsImplicitOrExplicitExpression()
                 //
                 return owner.AncestorsAndSelf().Any(n => n is CSharpImplicitExpressionSyntax || n is CSharpExplicitExpressionSyntax);
             }
+
+            bool IsInSectionDirective()
```

Comment:
```suggestion
            static bool IsInSectionDirective()
```

Reply:
Actually I can't do this as I'm using a method variable `owner`